sites/public/src/forms/finder/FinderRentalCosts.tsx

```diff
@@ -0,0 +1,67 @@
+import { FrontendListingFilterStateKeys } from "@bloom-housing/shared-helpers"
+import { Field, t } from "@bloom-housing/ui-components"
+import { UseFormMethods } from "react-hook-form"
+import { FinderQuestion } from "../../../pages/finder"
+
+const FinderRentalCosts = (props: {
```

Comment:
If you want to go down the route where you don't have to pass in all of these props, you could switch to the `useFormContext` style. https://react-hook-form.com/api/useformcontext/. With a fairly small form this isn't necessary though. But if more questions (non multi-select ones) we might want to think to moving to that style

Reply:
Sounds good! I'll keep this in mind going forward